docs/modules/deploy/pages/updating-license-rest.adoc

```diff
@@ -1,17 +1,17 @@
-= Updating a License Key with the REST API
-:description: The REST API has a /license endpoint that allows you to temporarily update the license on a running member.
+= Updating a License Key at Runtime with the REST API
+:description: You can use the /license endpoint of the REST API to update the license on a running member.
 
-The REST API has a `/license` endpoint that allows you to temporarily update the license on a running member.
-
-WARNING: If a member shuts down, its license key will not be persisted. To persist the license key, you can add the license key to a configuration file.
+You can use the `/license` endpoint of the REST API to update the license on a running member.
```

Comment:
Duplicated?

Reply:
The `description` attribute is used for the HTML meta-description. Although we can include the description on the page (`{description}`), when we do this the inline styles are not processed and we end up with unformatted code.